activerecord/lib/active_record/associations/preloader.rb

```diff
@@ -83,7 +83,7 @@ class Preloader # :nodoc:
       #   book's author, as well as that author's avatar.
       #
       # +:associations+ has the same format as the +:include+ method in
-      # <tt>ActiveRecord::QueryMethods</tt>. So +associations+ could look like this:
+      # +ActiveRecord::QueryMethods+. So +associations+ could look like this:
```

Comment:
I think we should link to include(s?) here (it won't let me do a real suggestion 😞 )

ActiveRecord::QueryMethods#includes

Reply:
Also had a TODO to follow up on this one, so funny we both saw that 😂